deps/rabbitmq_stream/test/stream_test_utils.erl

```diff
@@ -0,0 +1,139 @@
+%% FIXME this clause is copied from protocol_interop_SUITE.erl
```

Comment:
please advise on what header should be used in this new file

Reply:
The same thing as most others, [like `rabbit`](https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbit/src/rabbit.erl).

```
%% This Source Code Form is subject to the terms of the Mozilla Public
%% License, v. 2.0. If a copy of the MPL was not distributed with this
%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
%%
%% Copyright (c) 2007-2024 Broadcom. All Rights Reserved. The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries. All rights reserved.
%%
```